medusa/db.py

```diff
@@ -491,7 +492,10 @@ def addColumn(self, table, column, column_type='NUMERIC', default=0):
         self.connection.action('UPDATE [%s] SET %s = ?' % (table, column), (default,))
 
     def checkDBVersion(self):
-        return self.connection.checkDBVersion()
+        if six.PY3:
+            return self.connection.checkDBVersion()[0]
```

Comment:
This should be `return self.connection.checkDBVersion()[0]` for both now. It will be evident once you rebase the PR.